extensions/amp-story/0.1/amp-story-desktop-user-overridable.css

```diff
@@ -0,0 +1,13 @@
+.i-amphtml-story-background {
+  background-color: transparent;
+}
+
+[desktop] amp-story-page {
```

Comment:
I don't actually think the current plan is to let any of the things from here down be overridden

Reply:
Done